annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -744,7 +748,8 @@ private TranscriptConsequence transcriptWithMostSevereConsequence(List<Transcrip
 
     private String getVariantClassificationFromMap(String variant) {
         String refAllele = mRecord.getREFERENCE_ALLELE().equals("-") ? "" : mRecord.getREFERENCE_ALLELE();
-        String tumorSeqAllele = getTumorSeqAllele(mRecord).equals("-") ? "" : getTumorSeqAllele(mRecord);
+        String tumorSeqAllele = MafUtil.resolveTumorSeqAllele(mRecord.getREFERENCE_ALLELE(), mRecord.getTUMOR_SEQ_ALLELE1(), mRecord.getTUMOR_SEQ_ALLELE2());
+        if (tumorSeqAllele.equals("-")) tumorSeqAllele = "";
```

Comment:
My interpretation of this is: we are dealing with a deletion and we are standardizing the representation to remove the placeholder character for a gap ("-"). My comment above regarding the possible problem with  .equals("-")  applies here ... unless all common flanking nucleotides are always stripped.